server/server.js

```diff
@@ -0,0 +1,42 @@
+import dotenv from "dotenv";
+import { Configuration, OpenAIApi } from "openai";
+
+dotenv.config();
+const config = new Configuration({
+  // apiKey: process.env.OPEN_AI_API_KEY
+  apiKey: "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"
+});
+const openai = new OpenAIApi(config);
+const runPrompt = async (question) => {
+  const prompt = question;
+  const response = await openai.createCompletion({
+    model: "text-davinci-003",
+    prompt: prompt,
+    max_tokens: 2048,
+    temperature: 1
+  });
+  const parsableJSONresponse = response.data.choices[0].text;
+  if (
+    question.includes("from english to hebrew") ||
+    question.includes("from english to arabic")
+  ) {
+    return parsableJSONresponse.split("").reverse().join("");
+  } else {
+    return parsableJSONresponse;
+  }
+};
+const question_EN_HE = `translate from english to hebrew : "Hello World 1"`;
```

Comment:
remove all of the questions - you used it for testing, its not needed in the product